stdlib/Test/src/Test.jl

```diff
@@ -685,7 +685,26 @@ with this macro. Use [`@test_logs`](@ref) instead.
 """
 macro test_nowarn(expr)
     quote
-        @test_warn r"^(?!.)"s $(esc(expr))
+        # Duplicate some code from `@test_warn` to allow printing the content of
+        # `stderr` again to `stderr` here while suppressing it for `@test_warn`.
+        # If that shouldn't be used, it would be possible to just use
+        #     @test_warn r"^(?!.)"s $(esc(expr))
```

Comment:
I think the empty regex may be more memorable
```suggestion
        #     @test_warn r"^$" $(esc(expr))
```
or perhaps even more simply:
```suggestion
        #     @test_warn isempty $(esc(expr))
```